docs/modules/ROOT/partials/label-taint-storage-nodes.adoc

```diff
@@ -1,14 +1,14 @@
 [source,bash,subs="attributes"]
 ----
-kubectl --as=cluster-admin label --overwrite node -lnode-role.kubernetes.io/worker \
+kubectl label --overwrite node -lnode-role.kubernetes.io/worker \
```

Comment:
This change only applies for the install how-to. As is, this will break all other Exoscale how-tos which create new storage nodes.

Instead the partial should use the `{kubectl_extra_args}` page attribute:

```suggestion
kubectl {kubectl_extra_args} label --overwrite node -lnode-role.kubernetes.io/worker \
```

Also add the `kubectl_extra_args` page attribute with empty content in `docs/modules/ROOT/pages/how-tos/exoscale/install.adoc` after `:provider: exoscale`:

```asciidoc
:kubectl_extra_args:
```